packages/webdriver-utils/src/metadata/commonMetaDataResolver.js

```diff
@@ -0,0 +1,14 @@
+import CommonDesktopMetaDataResolver from './commonDesktopMetaDataResolver.js';
```

Comment:
why are these classes prefixed with `Common` ? I understand that its a base class, but you can use `DesktopMetaDataResolver` and subclass as `MacDesktopMetaDataResolver` etc.

Also note: `Resolver` is a pattern that takes a state and resolves correct handler for the state. For example `ProviderResolver` takes  driver and returns a class instance that can handle that state. 

here the class name should be just `DesktopMetadata` [ and you can have a `MetadataResolver` that takes driver, looks at caps and returns correct metadata class.